### PR TITLE
Remove support for 'Windows 7'.

### PR DESCRIPTION
**Pull request by tomkirbygreen**
Remove support for Windows 7 but leave the whole question of .NET Framework vs .NET Core and .NET Standard 2.0 alone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -383,18 +364,6 @@ Token requests are issued by your servers and signed using your private API key.
...
-When a 128-bit or 256-bit key is provided to the library, all payloads are encrypted and decrypted automatically using that key on the channel. The secret key is never transmitted to Ably and thus it is the developer's responsibility to distribute a secret key to both publishers and subscribers.
-
-```csharp
-var secret = Crypto.GetRandomKey();
-IRealtimeChannel encryptedChannel = client.Channels.Get("encryptedChannel", new ChannelOptions(secret));
-await encryptedChannel.PublishAsync("name", "sensitive data"); // Data will be encrypted before publish
-var history = await encryptedChannel.HistoryAsync();
-var data = history.First().data; // "sensitive data" the message will be automatically decrypted once received
-```
-
```
Comment (AndyNicks) [inline README.md:397]: What was the reason to remove this part? .Net SDK Supports encryption, doesn't it?

Comment (tomkirbygreen) [reply README.md:397]: Removed as it's effectively repeating  an existing code snippet.

**Review by AndyNicks — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -11,87 +11,68 @@ This is a .NET client library for Ably. The library currently targets the [Ably
...
-## Unsupported platforms
-
-### .NET 6 and Visual Studio 2022
-
-`.NET 6` and `Visual Studio 2022`, both currently in preview, at not supported at this time but will be supported when they are officially released.
-
-### Portable Class Libraries
-
-A Portable Class Library version is not available. Portable Class Libraries are now considered deprecated and shared code should be packaged in .NET Standard Libraries. More information can be found [here](https://docs.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/code-sharing).
+The library creates a number of threads and all callbacks are executed on non UI threads. This makes it difficult to update UI elements inside any callback executed by Ably. To make it easier we support capturing the `SynchronizationContext` and synchronizing callbacks to the UI thread.
```
Comment (marto-stadion) [inline README.md:41]: Currently `CaptureCurrentSynchronizationContext` is marked as obsolete. Should we remove the obsolete attribute if we are including it in the readme?

**Review by marto-stadion — COMMENTED** [review]
a couple of small things

File: README.md [hunk 3]
```diff
@@ -11,87 +11,68 @@ This is a .NET client library for Ably. The library currently targets the [Ably
 
 ## Supported platforms
 
-* .NET 4.6.2+ &ast;
-* .NET Core &ast;&ast;
+* .NET (Core) 3.1+
+* .NET Framework 4.8
```
[the remark this answers is not in the document]
Comment (tomkirbygreen) [reply README.md:15]: I'd prefer to keep OS support and .NET versions separate. I originally had both in this PR but then thought that they're not necessarily joined-at-the-hip.

**Review by marto83 — APPROVED** [review]
Looks good

**Pull request merged**